app/controllers/replies_controller.rb

```diff
@@ -97,4 +97,8 @@ def find_new_thread
       @new_thread = NewThread.friendly.find_by_id(params[:new_thread_id])
     end
   end
+
+  def friendly_url
+    @new_thread = NewThread.friendly.find(params[:new_thread_id])
+  end
 end
```

Comment:
All I am saying is that if you are applying authorization on reply controller then ability would need to know about `@reply` variable. And that should be done before call to `authorize_resource`. That was the function of `load_resource` since you are not using it (in this case, I don't see why), you will have to find a way to set that variable before you make call to ability.